cmd/directory.go

```diff
@@ -45,14 +45,23 @@ func runDirectory(cmd *cobra.Command, args []string) {
 
 	detector := Detector(cmd, cfg, source)
 
+	// set follow symlinks flag
+	if detector.FollowSymlinks, err = cmd.Flags().GetBool("follow-symlinks"); err != nil {
+		log.Fatal().Err(err).Msg("")
+	}
```

Comment:
Likewise, the code to set `detector.FollowSymlinks` wasn't included in this command.